app/controllers/application_controller.rb

```diff
@@ -1,4 +1,5 @@
 class ApplicationController < ActionController::API
   include Response
   include ExceptionHandler
+  include ActionController::HttpAuthentication::Basic::ControllerMethods
```

Comment:
Recommendation for Refactoring: If you make this a plain old Ruby class, you can get rid of this line

Reply:
It looks like line 4 of the items controller is using this. 
`http_basic_authenticate_with name: Rails.application.credentials.username, password: Rails.application.credentials.password, only: :create`

Reply:
Ah, nevermind then.